static/js/app/modal.js

```diff
@@ -0,0 +1,17 @@
+window.app.modal = function() {
+  const modal = document.querySelector('.p-modal');
+  const modalTrigger = document.querySelector('[data-js=modalTrigger]');
+
+  window.onload = function() {
```

Comment:
why is the class `u-hide` required if we're just going to remove it on page load?

Also defining the onload like this is not scalable. It'll overwrite any other previously defined onload function from other files.